ibis/expr/datatypes.py

```diff
@@ -83,7 +83,9 @@ def __repr__(self) -> str:
         )
 
     def __str__(self) -> str:
-        return self.name.lower()
+        return '{}{}'.format(
```

Comment:
instead of this you can simply define
```
__str__ = __repr__
```
as repr already does this.

Reply:
actually they are not the same.

with the current approach it prints:

```
ibis.Schema {  
  foo  int64
  bar  int64[non-nullable]
  baz  boolean
}
```

using __str__ = __repr__, it prints:

```
ibis.Schema {  
  foo  Int64(nullable=True)
  bar  Int64(nullable=False)
  baz  Boolean(nullable=True)
}
```